.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# PULL REQUEST TEMPLATE
+
+## Description
+> Please provide a summary of your changes here. 
+> * Which issue is this pull request belonging to? (*Refer to issue here*)
+> * How is it solving the issue?
+> * Did you add any new dependencies that are required for your change?
+
+## Request reviewer
+> You can add desired reviewers here with an @mention.  
+> For a full list of reviewers check out the *Committers* slot [here](https://projects.eclipse.org/projects/technology.sw360.antenna/who)
+
+## Type of Change
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Improvements
+- [ ] Documentation update
+- [ ] Other
+
+## How Has This Been Tested?
+- [ ] Specific Unit Test
+- [ ] Executing the Example Projects
+- [ ] `mvn test` of whole project
+- [ ] `mvn test` of submodule
+- [ ] `gradle test` (for the gradle plugin)
+
+## Checklist
+- [ ] My code follows the style and guidlines of this project
+- [ ] I have self-reviewed my code 
+- [ ] I have provided tests that prove my change is working 
+- [ ] Existing tests still pass with my changes 
+- [ ] I have updated the documentation accordingly to my changes
+- [ ] My changes generate no new warnings
```

Comment:
IMO this is something that should be technically enforced.

Reply:
How would that be done? As a check run when doing a PR? Or is that also something github actions does?
And how would you technically enforce that the documentation is changed or that test have been provided? 
The tests that are still passing is obviously controlled. And checking for warnings is also doable.

Reply:
I was only referring to the "My changes generate no new warnings" line, and I'd assume there's a way to make Maven fail a build (and thus CI) if a threshold of *X* warnings is exceeded (if you talk about compiler warnings; but similarly for warnings from static code analysis).